notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb

```diff
@@ -18,6 +18,8 @@
     "- [Original Pipeline Inference](#FastSAM-in-Ultralytics)\n",
```

Comment:
<p>Could you please add a reference to this example <a href="https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/openvino/yolov8" target="_blank">https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/openvino/yolov8</a> since the quantization code was taken from there?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>added a mention </p>